### PR TITLE
Rename main branch to c4t

### PR DESCRIPTION
**Pull request by mo-c4t**
replace foundation with network domain
deleting old workflows for creating dockers
renaming main branch to c4t

### CODE REVIEW

File: .github/workflows/update_gcs_buckets.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+        uses: 'google-github-actions/auth@v1'
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v1'
+
+      - name: update bucket files
+        run: gsutil rsync -R -d dist gs://${{ steps.setBucketNames.outputs.bucket }}
```
Comment (havan) [inline .github/workflows/update_gcs_buckets.yml]: Here parallel synchronization would speeds up the rsync significantly. We used that for docs:
```yaml
run: gsutil -m rsync -R -d dist gs://${{ steps.setBucketNames.outputs.bucket }}
```

File: src/utils/route-paths.ts [hunk 2]
```diff
@@ -28,14 +28,14 @@ export const VALIDATORS = '/validators';
 
 export const WALLET = 'https://wallet.camino.network/';
 
-export const DOCS = 'https://docs.camino.foundation/';
+export const DOCS = 'https://docs.camino.network/';
 
 export const DISCORD = 'https://discord.gg/camino';
 
 export const TWITTER = 'https://twitter.com/caminonetwork';
```
Comment (havan) [inline src/utils/route-paths.ts]: This URL is wrong. It leads to a suspended account. The correct one should be https://twitter.com/camino_network/
We should update it while we're at it.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      matrix:
+        node-version: [16]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: yarn
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install
+      - run: yarn build
```
Comment (havan) [inline .github/workflows/build.yml:23]: Should we use lint here?

**Review by havan — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 3 above)
Comment (mo-c4t) [reply .github/workflows/build.yml:23]: Done

File: src/utils/route-paths.ts (hunk 2 above)
Comment (mo-c4t) [reply src/utils/route-paths.ts]: Done

File: .github/workflows/update_gcs_buckets.yml (hunk 1 above)
Comment (mo-c4t) [reply .github/workflows/update_gcs_buckets.yml]: Done

**Review by Chriserus — COMMENTED** [review]
Is yarn.lock in this PR intentionally?

**Comment by mo-c4t** [comment]
> Is yarn.lock in this PR intentionally?

reverted

**Review by Chriserus — APPROVED** [review]

**Review by havan — APPROVED** [review]
LGTM

**Pull request merged**